crates/viewer/re_data_ui/src/blob.rs

```diff
@@ -142,3 +157,46 @@ pub fn blob_preview_and_save_ui(
         });
     }
 }
+
+fn show_video_blob_info(
+    ui: &mut egui::Ui,
+    ui_layout: UiLayout,
+    video_result: &Result<re_renderer::video::Video, re_renderer::video::VideoError>,
+) {
+    match video_result {
+        Ok(video) => {
+            if ui_layout.is_single_line() {
+                return;
+            }
+
+            re_ui::list_item::list_item_scope(ui, "video_blob_info", |ui| {
+                ui.list_item_flat_noninteractive(re_ui::list_item::LabelContent::new(
+                    "Video properties",
+                ));
+                ui.list_item_flat_noninteractive(
+                    PropertyContent::new("Dimensions").value_text(format!(
+                        "{}x{}",
+                        video.width(),
+                        video.height()
+                    )),
+                );
+                ui.list_item_flat_noninteractive(PropertyContent::new("Duration").value_text(
+                    format!(
+                        "{}",
+                        re_log_types::Duration::from_millis(video.duration_ms() as _)
```

Comment:
Can we also show the number of frames in the video?

Reply:
I actually had this in for a moment but then removed it again because it requires iterating over all the samples of a video. Which isn't terribly expensive and could be cached, but then I saw that video players only do this upon read and usually carefully talk about samples, not frames 🤔
@jprochazk what do you think?

Reply:
putting it in for now, but calling it `Sample count` instead. I figure the contention with "frames" is that it may refer to full frames/iframes.

Reply:
Yeah, I would be careful about this. Some people may think that `num_frames / duration = fps`, but that's not true, videos may have variable frame rate. At the same time, we don't want to overload users with video codec/container specific stuff that they have to understand, and for all intents and purposes one sample = one frame.